protoc_plugin/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 18.0.3
```

Comment:
This is actually a breaking change since it requires an updated protobuf version

Reply:
I thought to publish 
https://github.com/dart-lang/protobuf/pull/294
and
https://github.com/dart-lang/protobuf/pull/290
and
https://github.com/dart-lang/protobuf/pull/282

Together as 19.0.0 to make fewer breaking revisions. This could be part of that.

Reply:
That sounds reasonable. How do you want go about merging them?